cristin-import/src/test/java/no/unit/nva/cristin/CristinDataGenerator.java

```diff
@@ -542,6 +542,10 @@ private static CristinContributorRoleCode randomCristinContributorRoleCode() {
         return randomArrayElement(CristinContributorRoleCode.values(), USE_WHOLE_ARRAY);
     }
 
+    private static Integer threeDigitPositiveNumberKnownInstitution() {
+        return RANDOM.nextInt(1, 1000);
+    }
+
```

Comment:
Dette er fiksen for å få fuzzy testen til å virke. Men etter at jeg fikset den så jeg virkelig ikke behovet for å fortsette å ha den.